src/locales/en.json5

```diff
@@ -441,8 +441,7 @@
   "mixin": {
     "request": {
       "alert": {
-        "fileSize": "The file “{name}” that you are trying to upload is larger than the 100 MB limit.",
-        "entityTooLarge":  "The data that you are trying to upload is too large.",
+        "fileSize": "The file “{name}” that you are trying to upload is larger than the 100 MB limit."
```

Comment:
This message is not used in src/util/request.js, so I'm leaving it where it is for now. I think it's very likely that we'll move it at some point in the future (probably as part of #675).